src/cmd-oscontainer

```diff
@@ -120,6 +124,8 @@ parser_build.add_argument("--from", help="Base image (default 'scratch')", defau
 parser_build.add_argument("src", help="OSTree repository")
 parser_build.add_argument("rev", help="OSTree ref (or revision)")
 parser_build.add_argument("name", help="Image name")
+parser_build.add_argument("--inspect-out", help="Write container JSON",
```

Comment:
Minor: maybe `image` instead of `container` to jive with the other help strings for the build subcommand?

Reply:
Also maybe add " to file" and `metavar=FILE`?